crowd_anki/representation/deck.py

```diff
@@ -171,7 +171,9 @@ def _save_deck(self, collection, parent_name):
         self.anki_dict = deck_dict
         self.anki_dict["name"] = full_name
         self.anki_dict["conf"] = self.metadata.deck_configs[self.deck_config_uuid].anki_dict["id"]
-        collection.decks.save()
+        collection.decks.save(deck_dict)
+        # TODO: remove when compatibility with Anki < 2.1.28 is no longer needed
+        # DeckManager.flush() is now a no-op
```

Comment:
As it's possible to specify versions for which we upload the plugin, I believe we can delete these straight away (same for other places)

Reply:
Done in 69b4ac1d68d7c273ec24a181f96f86cba5eabb36!